ownCloud/Server List/ServerListTableViewController.swift

```diff
@@ -388,3 +360,24 @@ class ServerListTableViewController: UITableViewController, Themeable {
 		OCBookmarkManager.shared.moveBookmark(from: UInt(fromIndexPath.row), to: UInt(to.row))
 	}
 }
+
+// MARK: - OCClassSettings support
+extension OCClassSettingsIdentifier {
+	static let app = OCClassSettingsIdentifier("app")
+}
+
+extension OCClassSettingsKey {
+	static let showBetaWarning = OCClassSettingsKey("show-beta-warning")
```

Comment:
We can document this in the `CONFIGURATION.md` file so if any contributor/developer/QA wants to test something and this beta pop-up appears they know how to hide it. Or at least document it in `README.md` or some other place.

Reply:
The next commit will amend `CONFIGURATION.md` accordingly.